Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/Indexer.kt

```diff
@@ -499,6 +499,19 @@ internal class NativeIndexImpl(val library: NativeLibrary, val verbose: Boolean
                 spelling = clang_getTypeSpelling(type).convertAndDispose().dropConstQualifier()
         )
 
+        CXType_Vector -> {
+            val elementType = convertType(clang_getElementType(type))
+            val size = clang_Type_getSizeOf(type)
+            val elemSize = clang_Type_getSizeOf(clang_getElementType(type))
+            val length = size / elemSize
+            val spelling = type.name // ex: __attribute__((__vector_size__(4 * sizeof(const float)))) const float
```

Comment:
`dropConstQualifier` hack may also be required here as above: https://github.com/JetBrains/kotlin-native/blob/e1b3fefd3333fa730bb00174b7eb24c90822f6a0/Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/Indexer.kt#L499

Reply:
dropConstQualifier won't do the trick, but we can generate spelling from the components